test/global_test_setup.clj

```diff
@@ -1,7 +1,11 @@
 (ns global-test-setup
   "This namespace's contents will be automatically loaded by `lein test` (or any test runner)."
   (:require
-   [refactor-nrepl.inlined-deps.toolsnamespace.v1v1v0.clojure.tools.namespace.repl :refer [set-refresh-dirs]]))
+   [clojure.tools.namespace.repl :refer [set-refresh-dirs]]))
```

Comment:
Maybe move this addition to the `catch`? Else an ambiguity is created

Reply:
👍